taccsite_cms/contrib/taccsite_sysmon/cms_plugins.py

```diff
@@ -0,0 +1,53 @@
+from cms.plugin_base import CMSPluginBase
+from cms.plugin_pool import plugin_pool
+from django.utils.translation import gettext_lazy as _
+
+from taccsite_cms.contrib.helpers import concat_classnames
+from taccsite_cms.contrib.taccsite_offset.cms_plugins import get_direction_classname
+
+from .models import TaccsiteSysmon
+
+@plugin_pool.register_plugin
+class TaccsiteSysmonPlugin(CMSPluginBase):
+    """
+    Components > "System Monitor" Plugin
+    https://url.to/docs/components/sysmon/
```

Comment:
I have been using this URL consistently (I think… so code reviews are good) for plugins that have no Confluence doc yet (would be within [… CMS UI - Components / Patterns](https://confluence.tacc.utexas.edu/pages/viewpage.action?pageId=117571624)).

<details>
<summary>Details</summary>

Ideally, the URL is an automatically generated instance of the plugin in use on Core CMS, but that feature does not exist, yet (a [manual dev version](https://dev.cep.tacc.utexas.edu/design-system/pattern-library/) is available, and I have a more populated version on my local database).

</details>